tests/functional/test_cache.py

```diff
@@ -0,0 +1,69 @@
+import os
+import shutil
+
+
+def _cache_dir(script):
+    results = script.run(
+        'python', '-c',
+        'from pip._internal.locations import USER_CACHE_DIR;'
+        'print(USER_CACHE_DIR)'
+    )
+    return str(results.stdout).strip()
```

Comment:
```suggestion
    return result.stdout.strip()
```

Why was this wrapped with `str`?

Reply:
Unintentional leftover from my first attempt before I remembered that, uh, reading the docs is useful. :sweat_smile: